### ISSUE TITLE
DotcomShell:  Accessibility:  "skip to main content link" is not working

### ISSUE THREAD

**Issue by Palke** [carbon-design-system/ibm-dotcom-library#2664]
Using basic TAB navigation within DotcomShell activates "skip to main content link" when in focus, but using key **enter** doesn't change focus to the main content - nothing happens.

There is also no element with id "#main-content" in DotcomShell.

Steps to replicate:
1) Open https://ibmdotcom-react.mybluemix.net/iframe.html?id=components-dotcom-shell--default&viewMode=story in new tab
2) Use TAB to navigate within the content until "skip to main content link" appears
3) Press enter

image
<img width="707" alt="image" src="https://user-images.githubusercontent.com/54019675/83799223-28f6ba80-a6ae-11ea-97d2-f3a448780dc1.png">

Expected:
Focus should be moved into the main content area and skip Masthead navigation.

v1.7.0
